packages/flutter_test/test/bindings_test.dart

```diff
@@ -20,15 +22,32 @@ void main() {
 
   group(AutomatedTestWidgetsFlutterBinding, () {
     test('allows setting defaultTestTimeout to 5 minutes', () {
-      final AutomatedTestWidgetsFlutterBinding binding = AutomatedTestWidgetsFlutterBinding();
       binding.defaultTestTimeout = const test_package.Timeout(Duration(minutes: 5));
       expect(binding.defaultTestTimeout.duration, const Duration(minutes: 5));
     });
   });
 
+  // The next three tests must run in order -- first using `test`, then `testWidgets`, then `test` again.
+
+  int order = 0;
```

Comment:
Nit: Alternatively you could use a `group` and do the incrementing in a `setUp`.  Fine either way though.

Reply:
I still live in hope that we can destroy `group` and `setUp` and so on...

Reply:
(i think it's way more readable if you just have the code running bit by bit rather than having to know what order setUp/group/etc run in, and what the implications of all those methods are, Zones, how it impacts futures, etc)

Reply:
Ah I hadn't realized anyone was anti-group/setUp, I've been using them just because they were there.  I'll think about this next time I encounter it.